packages/cli/src/oclif/commands/integrations.js

```diff
@@ -3,11 +3,16 @@ const { buildFlags } = require('../buildFlags');
 
 const { listApps } = require('../../utils/api');
 
-class AppsCommand extends BaseCommand {
+class Integrations extends BaseCommand {
   async perform() {
+    if (this.argv.includes('apps')) {
+      this.warn('The `apps` command is deprecated. Use `integrations` instead');
```

Comment:
This also appears in `hooks/deprecated.js`. Do we still need it here?

Reply:
ah, it appears there because it doesn't work here. `this.id` and `this.args` always use the actual version, not the alias. Will remove.